src/it/resources/org/checkstyle/suppressionxpathfilter/matchxpath/SuppressionXpathRegressionMatchXpathEncodedFive.java

```diff
@@ -0,0 +1,26 @@
+////////////////////////////////////////////////////////////////////////////////
+// checkstyle: Checks Java source code for adherence to a set of rules.
+// Copyright (C) 2001-2021 the original author or authors.
+//
+// This library is free software; you can redistribute it and/or
+// modify it under the terms of the GNU Lesser General Public
+// License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+//
+// This library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public
+// License along with this library; if not, write to the Free Software
+// Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+////////////////////////////////////////////////////////////////////////////////
+
+package org.checkstyle.suppressionxpathfilter.matchxpath;
+
+public class SuppressionXpathRegressionMatchXpathEncodedFive {
+
+    String ampersandChar = "&testThree" + // warning
```

Comment:
please put in name of class target symbol you try to validate - "Amp".

please apply this rule to all other files that have in name "One, Two, Three, .... Five". Lets have names self descriptive

Reply:
Done

Reply:
Even I don't know but I didn't change it since it was mentioned in [XMLLogger](https://github.com/checkstyle/checkstyle/blob/5f53bfe41e96a0fcd0a8266ffc0db0fb32c54d6e/src/main/java/com/puppycrawl/tools/checkstyle/XMLLogger.java#L275) for the exact same thing, so I didn't change the description. I had changed it now.